app/views/filters/index.html.erb

```diff
@@ -43,7 +43,7 @@ end %>
           </td>
         <% end %>
         <td>
-          <%= _(filter.resource_class.try(:humanize_class_name) || filter.resource_type || '(Miscellaneous)') %>
+          <%= _(filter.resource_class.try(:humanize_class_name) || filter.resource_type_label ) %>
```

Comment:
Should the `resource_type_label` already try to humanize or will that just end up wrong in the captions?

Reply:
Yes, thanks